docs/badge.md

```diff
@@ -57,6 +64,22 @@ import { Avatar, Badge } from 'react-native-elements'
     containerStyle={{ position: 'absolute', top: -4, right: -4 }}
   />
 </View>
+
+const BadgedIcon = withBadge(1)(Icon)
+<BadgedIcon type="ionicon" name="ios-chatbubbles" />
+
+// Using the decorator proposal
+@connect(state => ({
```

Comment:
Not sure we should add this here, I assure you we'll get issues asking why my code doesn't work.

Stick to standard aAPIs

Reply:
Actually I suggested @janhesters to add this as it is a pretty common use case for a decorator

Reply:
Decorators aren't apart of the spec, this is advanced usage. Someone that's already knowledgeable of decorators would already know that they can use the hoc in that way, so this would only confuse beginners(newer users).

Reply:
So if you don't show how to use it in the `Usage` section, where do you explain to the user how to use this decorator ?

Reply:
Any news on this?